delegation-toolkit/get-started/delegation-quickstart.md

```diff
@@ -43,9 +48,11 @@ const bundlerClient = createBundlerClient({
 
 ### 3. Create a delegator account
 
-[Create a delegator smart account](../how-to/create-smart-account/index.md) to set up a delegation.
+Create an account to represent Alice, the delegator who will set up the delegation.
+The delegator must be a [smart account](../how-to/create-smart-account/index.md).
 
-This example configures a [Hybrid](../how-to/create-smart-account/configure-accounts-signers.md#configure-a-hybrid-smart-account) delegator account:
+This example configures a [Hybrid](../concepts/smart-accounts.md#hybrid-smart-account) smart account,
+which is a flexible smart account implementation that supports both an externally owned account (EOA) owner and any number of P256 (passkey) signers:
```

Comment:
For the sake of keeping terminology same everywhere, maybe we can use `WebAuthn` instead of `P256`?